samples/functionapp-csharp/README.md

```diff
@@ -0,0 +1,45 @@
+# Azure Function App with Durable Client Dependency Injection
+
+This project contains an Azure Function App that calls a Durable Function through a Durable Client dependency injection.
+
+## Local setup
+
+Create a local.settings.json file and add two properties: a `Storage` property and a `TaskHub` property. Add the storage account connection string that you are using for the Durable Function. Having the same storage account allows the two function apps to communicate.
+
+```json
```

Comment:
We should consider shipping with a default value for local.settings.json here (but have it explicitly git-ignored) so that customers just have to tweak the file rather than creating it from scratch.

Reply:
Should I just create a local.settings.json file with default values then? The precompiled samples folder includes a local.settings.json file so it looks like it's already included in the .gitignore.

Reply:
Yeah, you can just copy what you have in this json blob right here and put that in the local.settings.json. Leave the placeholders as is in the json, and just remark here in the docs that customers should configure those values before running the samples.